src/util/hass-translation.ts

```diff
@@ -64,12 +65,12 @@ export function getActiveTranslation() {
 // when DOM is created in Polymer. Even a cache lookup creates noticeable latency.
 const translations = {};
 
-export function getTranslation(fragment?, translationInput?) {
-  const translation = translationInput || getActiveTranslation();
+export async function getTranslation(hass?, fragment?, translationInput?) {
```

Comment:
I think that `hass` should not be optional here, or we should pass in the language directly. Decouple the "how to get the language" from the "loading resources"

Reply:
Done